.travis.yml

```diff
@@ -1,24 +0,0 @@
-language: cpp
```

Comment:
Should I keep this file as is for now? I saw you were trying to use it for osx testing...